storage/remote/influxdb/client.go

```diff
@@ -14,148 +14,97 @@
 package influxdb
 
 import (
-	"bytes"
-	"encoding/json"
-	"fmt"
-	"io/ioutil"
 	"math"
-	"net/http"
-	"net/url"
-	"time"
 
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/common/model"
 	"github.com/prometheus/log"
 
-	"github.com/prometheus/prometheus/util/httputil"
-)
-
-const (
-	writeEndpoint   = "/write"
-	contentTypeJSON = "application/json"
+	idb_client "github.com/influxdb/influxdb/client"
```

Comment:
How about `influxclient`?

The underscore looks sooo weird in Go code... (not feeling strongly, though... if golint is fine with it...)